banyand/metadata/embeddedserver/server.go

```diff
@@ -103,3 +139,37 @@ func NewService(_ context.Context) (metadata.Service, error) {
 	}
 	return s, nil
 }
+
+func (s *server) RegisterDefrag() {
```

Comment:
```suggestion
func (s *server) registerDefrag() {
```

Please don't expose this function.